test/integration/vhds_integration_test.cc

```diff
@@ -429,7 +161,7 @@ TEST_P(VhdsIntegrationTest, RdsUpdateWithoutVHDSChangesDoesNotRestartVHDS) {
 //  - Upstream makes a request to an (now) unknown domain
 //  - A VHDS DiscoveryResponse received containing update for the domain
 //  - Upstream receives a 200 response
-TEST_P(VhdsIntegrationTest, VhdsVirtualHostAddUpdateRemove) {
+TEST_P(VhdsIntegrationTest, DISABLED_VhdsVirtualHostAddUpdateRemove) {
```

Comment:
Why are tests also being disabled?

Reply:
whoops - I moved them over and forgot to delete them from the old test file - ditto the commented out tests